test/utils/wasm_engine.hpp

```diff
@@ -31,11 +31,9 @@ class WasmEngine
     /// Consecutive invocations replace the internal module and invalidate the internal instance.
     virtual bool parse(bytes_view input) = 0;
 
-    /// Instantiates the internal module creating an internal instance.
+    /// Instantiates the internal module from the wasm binary input (parsing included).
     /// Returns false on instantiation error.
```

Comment:
This is somewhat incorrect now.

Reply:
Looking at the final diff, this is the only nit left.

Reply:
Fixed.